airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -270,6 +274,7 @@ def __init__(
         reattach_on_restart: bool = True,
         startup_timeout_seconds: int = 120,
         get_logs: bool = True,
+        container_logs: list[str] | str | bool = BASE_CONTAINER_NAME,
```

Comment:
Does this accept `False`? If not, this should be `Literal[True]` instead of `bool`.

Reply:
Good point. Yeah it doesn't accept False. Only True